discordrb.json

```diff
@@ -57,6 +57,28 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "userLoginWithPassword": "yes",
```

Comment:
Whether the lib supports logging in as a user with email + password (`/api/login`)